lib/locales/hu.ts

```diff
@@ -0,0 +1,938 @@
+    pirmin_of_hornbach_abbot: 'Saint Pirmin, Abbot',
+    pius_francesco_forgione_priest: 'Saint Pius of Pietrelcina, Priest',*/
+    pius_ix_pope: 'Boldog IX. Piusz pápa',
+    pius_v_pope: 'Szent V. Piusz pápa',
+    pius_x_pope: 'Szent X. Piusz pápa',
+    /*polycarp_of_smyrna_bishop: 'Saint Polycarp, Bishop and Martyr',
+    pontian_i_pope_and_hippolytus_of_rome_priest: 'Saints Pontian, Pope, and Hippolytus, Priest, Martyrs',
+    pothinus_of_lyon_bishop_blandina_of_lyon_virgin_and_companions_martyrs: 'Saints Pothinus, Bishop, Blandina, Virgin, and Companions, Martyrs',
+    presentation_of_the_blessed_virgin_mary: 'The Presentation of the Blessed Virgin Mary',
+    presentation_of_the_lord: 'The Presentation of the Lord',
+    procopius_of_sazava_abbot: 'Saint Procopius of Sázava, Abbot',
+    publius_of_malta_bishop: 'Saint Publius, Bishop',
+    queenship_of_the_blessed_virgin_mary: 'The Queenship of the Blessed Virgin Mary',
+    quirinus_of_sescia_bishop: 'Saint Quirinus of Sescia, Bishop and Martyr',
+    rabanus_maurus_bishop: 'Saint Rabanus Maurus, Bishop',
+    radim_of_gniezno_bishop: 'Saint Radim, Bishop',
+    rafqa_pietra_choboq_ar_rayes_virgin: 'Saint Rafqa Pietra Choboq Ar-Rayès, Virgin',
+    raphael_chylinski_priest: 'Blessed Raphael Chyliński, Priest',
+    raphael_guizar_y_valencia_bishop: 'Saint Raphael Guízar y Valencia, Bishop',
+    raphael_of_saint_joseph_kalinowski_priest: 'Saint Raphael of Saint Joseph Kalinowski, Priest',
+    raymond_of_penyafort_priest: 'Saint Raymond of Penyafort, Priest',
+    remigius_of_reims_bishop: 'Saint Remigius, Bishop',
+    richard_gwyn_martyr: 'Saint Richard Gwyn, Martyr',
+    richard_of_chichester_bishop: 'Saint Richard of Chichester, Bishop',
+    richardis_of_swabia_empress: 'Saint Richardis, Empress',
+    rita_of_cascia_religious: 'Saint Rita of Cascia, Religious',
+    robert_bellarmine_bishop: 'Saint Robert Bellarmine, Bishop and Doctor of the Church',
+    roch_gonzalez_alphonsus_rodriguez_and_john_del_castillo_priests: 'Saints Roch González, Alphonsus Rodríguez and John del Castillo, Priests and Martyrs',
+    roch_of_montpellier: 'Saint Roch',
+    romuald_of_ravenna_abbot: 'Saint Romuald, Abbot',
+    rosalie_jeanne_marie_rendu_virgin: 'Blessed Rosalie Rendu, Virgin',
+    rose_of_lima_virgin: 'Saint Rose of Lima, Virgin',
+    rose_of_lima_virgin_copatroness_of_the_philippines: 'Saint Rose of Lima, Virgin, Copatroness of the Philippines',
+    rose_philippine_duchesne_virgin: 'Saint Rose Philippine Duchesne, Virgin',
+    rupert_of_salzburg_and_virgilius_of_salzburg_bishops: 'Saints Rupert and Virgilius of Salzburg, Bishops',
+    salomea_of_poland_religious: 'Blessed Salomea of Poland, Religious',
+    sancha_of_portugal_and_mafalda_of_portugal_virgins: 'Blessed Sancha and Mafalda, Virgins',*/
+    sara_salkahazi_virgin: 'Boldog Salkaházi Sára szűz és vértanú',
```

Comment:
According to [Direktórium 2022](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=104), there is a comma between `Sára` and `szűz`, however, it other celebration names, there is none. I personally miss the comma everywhere, however, if it is not used in the Hungarian translation of the Roman Missal, we should not use it anywhere.

In either case, we should be consistent: either use the comma everywhere or not, or find a rule when to use it and when not.

Reply:
I've checked it in *A magyar helyesírás szabályai* and I found that there shouldn't be a comma between the name and the title. Well this is not completely clear for me, because it says that if the name has an appellative (like *vértanú*) there is no need for the comma. But if you can inflect the name and the appellative as well there should be a comma. However, in Hungarian we don't say ~~*Szent Máténak, apostolnak*~~, we say instead *Szent Máté apostolnak*, we don't inflect the name. I don't know if *Boldog Salkaházi Sárának, szűz és vértanúnak* is correct but I would say *Boldog Salkaházi Sára szűz és vértanúnak*.

Reply:
Thank you for trying to find a solution to the issue! :pray:

As for the orthography rules, well, there has been (and probably are) some arguments between linguists and liturgists in Slovakia regarding a comma between a title like _Martyr_ and the following _and_: linguists say it is redundant, liturgists still use in (in both official liturgical books like the Roman Missal and in _Ordos_). I presume a similar issue might be in Hungarian too.

At first, I tried to use grammatically correct celebration names, later I shifted to use celebration names from official/primary sources when available, as I (now) consider the celebration names officially _assigned_ and thus it should not be changed (unless it contains a typo or the celebration name is officially changed, e.g. a title added or canonisation level changed to `Saint`). For celebrations that are not part of any calendar proper in a particular language (based on the region/country in which a particular language is officially used), I try to translate the celebration name in the same way as those celebrations that are officially translated.

Therefore I suggest to use celebration names as they are printed in the Hungarian translation of the Roman Missal as the first and primary source. If it uses commas between a personal name and its title(s), then use it. If not, don’t use it. If the Roman Missal sometimes use it and sometimes not, try to create a rule when to use it and when not, tell us about your findings (in order to let us document it) and abide those rules in all celebration names. :wink:

Just a note: that [_Direktórium 2022_](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf) contains many celebrations that are celebrated in Slovakia, but not in Hungary, which suggests the its editor (compiler) comes from or is somehow related to Slovakia, thus they does not necessarily know Hungarian perfectly (I might be wrong though), therefore they might made some mistakes when translating a Slovak celebration name (`Blahoslavenej Sáry Salkaháziovej, panny a mučenice`) to Hungarian (`Boldog Salkaházi Sára, szűz és vértanú`).

In the end, I think that in Hungarian the comma should not be used actually. [Here](https://www.salkahazisara.com/hu/liturgia-2/) this celebration is written without a comma between the personal name and the titles.

Thank you, again, for you time and willingness to help and improve romcal! :pray:

Reply:
You're welcome!